demo-ui/app/uk/gov/ons/addressIndex/demoui/modules/VersionModule.scala

```diff
@@ -38,12 +35,15 @@ class DemoUIAddressIndexVersionModule @Inject()(
   lazy val dataVersion: String = {
 
     Try(Await.result(
-      apiClient.versionQuery()
+      apiClient.versionQuery(tempApiKey)
         .map { resp: AddressResponseVersion =>
           resp.dataVersion
         }, 10 seconds)).getOrElse(dummyVersion)
   }
 
   val dummyVersion = "not found"
-
+  /** PLEASE Remember to modify module
+    * Add move to config -
+  */
+  val tempApiKey = "exxx-xxxxx-xxxxx-xxxxx-xxxxxx"
```

Comment:
Is this still relevant? I thought we would not require api key for version request.